spinalcordtoolbox/vertebrae/core.py

```diff
@@ -372,32 +418,33 @@ def compute_corr_3d(src, target, x, xshift, xsize, y, yshift, ysize, z, zshift,
     for iz in zrange:
         # if pattern extends towards the top part of the image, then crop and pad with zeros
         if z + iz + zsize + 1 > nz:
-            # sct.printv('iz='+str(iz)+': padding on top')
+            sct.printv('iz=' + str(iz) + ': padding on top')
             padding_size = z + iz + zsize + 1 - nz
-            data_chunk3d = src[x - xsize: x + xsize + 1,
-                               y + yshift - ysize: y + yshift + ysize + 1,
-                               z + iz - zsize: z + iz + zsize + 1 - padding_size]
+            data_chunk3d = src[xtarget - 10:xtarget + 10,
+                           y + yshift: y + yshift + ysize + 1,
+                           z + iz - zsize: z + iz + zsize + 1 - padding_size]
```

Comment:
Nitpick: indentation (here and for other data_chunk3d/data_chunk1d statements)

Reply:
done in 3fbbb20